app/models/bulk_upload.rb

```diff
@@ -0,0 +1,187 @@
+class BulkUpload
+  include ActiveModel::Model
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+
+  SPREADSHEET_CONTENT_TYPES = %w[
+    application/vnd.ms-excel
+    application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+  ].freeze
+
+  FIRST_DATA_ROW = 7
```

Comment:
Is the 7 likely to change or will it always be a constant?

Reply:
My assumption is it will be constant as long as we use this template because 1-6 are the template headers. TBC though.